tests/modeladmin/test_checks.py

```diff
@@ -84,6 +84,17 @@ class TestModelAdmin(ModelAdmin):
 
         self.assertIsValid(TestModelAdmin, ValidationTestModel)
 
+    def test_attname_mis_references(self):
+        class TestModelAdmin(ModelAdmin):
+            raw_id_fields = ('band_id',)
+
+        self.assertIsInvalid(
+            TestModelAdmin, ValidationTestModel,
+            "The value of 'raw_id_fields[0]' refers to 'band_id', "
+            "which is not an attribute of 'modeladmin.ValidationTestModel'.",
```

Comment:
This makes me wonder if we should change the message of the check to be
```suggestion
            "which is not a field of 'modeladmin.ValidationTestModel'.",
```
Any thoughts about that @felixxm?

Reply:
Agreed, we should also audit messages in other `admin` checks which refer to `attribute` such as `admin.E019`, `admin.E022`, etc. It looks we made a bulk update in 3c5fc708f1a8f60c05182869f6f3ec13697bbcf2.

Reply:
I will do this in a separate commit.